content/chat/rooms/index.textile

```diff
@@ -1,48 +1,54 @@
+A @room@ is created, or an existing one is retrieved from the @rooms@ collection using the "@rooms.get()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/Rooms.html#get method:
 
 ```[javascript]
 const room = chatClient.rooms.get('basketball-stream');
 ```
 
-When you create or retrieve a room using @rooms.get()@, you need to choose which additional chat features you want enabled in that room. This is configured by passing a "@RoomOptions@":/_LINK object as the second argument. In addition to setting which features are enabled, @RoomOptions@ also configures the properties of the associated features, such as the timeout period for typing indicators.
+When you create or retrieve a room using @rooms.get()@, you need to choose which additional chat features you want enabled for that room. This is configured by passing a "@RoomOptions@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/RoomOptions.html object as the second argument. In addition to setting which features are enabled, @RoomOptions@ also configures the properties of the associated features, such as the timeout period for typing indicators.
 
 ```[javascript]
-const room = chatClient.rooms.get('basketball-stream', {PresenceOptions, ReactionsOptions});
+const room = chatClient.rooms.get('basketball-stream', {presence, reactions, typing, occupancy});
 ```
 
-You can also use a property for each @RoomOption@ to configure whether the default settings should be used:
+You can also use the @RoomOptionsDefaults@ property for each @RoomOption@ to configure whether the default settings should be used:
 
 ```[javascript]
-const room = chatClient.rooms.get('basketball-stream', {TypingOptions: DefaultTypingOptions});
+const room = chatClient.rooms.get('basketball-stream', {presence: RoomOptionsDefaults.presence});
 ```
 
-Enable each feature using its associated option:
+Or configure each feature using your own values:
+
+```[javascript]
+const room = chatClient.rooms.get('basketball-stream', {typing: {timeoutMs: 5000}});
+```
+
+Enable each feature using its associated option. The details of the options available to each feature are documented on their respective pages:
 
 | Feature | @RoomOption@ | Default settings |
-| "Presence":/chat/rooms/presence | @PresenceOptions@ | @DefaultPresenceOptions@ |
-| "Occupancy":/chat/rooms/occupancy | @OccupancyOptions@ | None |
-| "Typing indicators":/chat/rooms/typing | @TypingOptions@ | @DefaultTypingOptions@ |
-| "Room reactions":/chat/rooms/reactions | @ReactionsOptions@ | None |
+| "Presence":/chat/rooms/presence | @presence@ | @RoomOptionsDefaults.presence@ |
+| "Occupancy":/chat/rooms/occupancy | @occupancy@ | None |
+| "Typing indicators":/chat/rooms/typing | @typing@ | @RoomOptionsDefaults.typing@ |
+| "Room reactions":/chat/rooms/reactions | @reactions@ | None |
```

Comment:
We should use `@RoomOptionsDefaults.reactions@` and `@RoomOptionsDefaults.occupancy@` even though they're empty.